runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectRunner.java

```diff
@@ -384,7 +384,7 @@ public State getState() {
       Collection<PTransform<?, ?>> steps = aggregatorSteps.get(aggregator);
       final Map<String, T> stepValues = new HashMap<>();
       for (AppliedPTransform<?, ?, ?> transform : evaluationContext.getSteps()) {
-        if (steps.contains(transform.getTransform())) {
+        if (steps != null && steps.contains(transform.getTransform())) {
```

Comment:
ok, in this case is it ok to return a `AggregatorValues` with empty `stepValues` map?